hergetto/deepl_ex
# [Feature]: Create basic translation functionality

## Type of change




# How Has This Been Tested?

1. Create a test application using `mix new`.
2. In the `mix.exs` file, import the `deepl_ex` package.
3. Create a new folder called `config`.
4. Inside the config folder, create a `config.exs`.
5. Write `use Config` at the top of the `config.exs` file.
6. Configure the application as stated in the documentation that can be generated using `mix docs`.
7. Run `iex -S mix`.
8. Inside iex run `DeeplEx.translate("Hoje vou comer.", :PT, :EN)`.
9. You should see `{:ok, "Today I'm going to eat."}`

- [x] These steps have been tested by a reviewer
